### PR TITLE
Replication job

### PR DESCRIPTION
**Pull request by Krishnan-Priyanshu**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

image
```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Unit tests
- [ x ] Acceptance tests

### CODE REVIEW

File: powerscale/provider/synciq_replication_job_resource.go [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+	time.Sleep(time.Duration(state.WaitTime.ValueInt64()) * time.Second)
+	tflog.Debug(ctx, "calling get syncIQ Replication Job on powerscale client")
+	readState, httpResp, err := helper.GetSyncIQReplicationJob(ctx, r.client, state.Id.ValueString())
+	if err != nil {
+		// need to discuss with team if we should remove resource from state or not
+		if httpResp != nil && httpResp.StatusCode == http.StatusNotFound {
```
Comment (shenda1) [inline powerscale/provider/synciq_replication_job_resource.go:171]: Any conclusion on this To-Do?

File: powerscale/provider/synciq_replication_job_resource.go [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+	if resp.Diagnostics.HasError() {
+		return
+	}
+	diags = req.Plan.Get(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+	if !plan.IsPaused.Equal(state.IsPaused) {
+		isPause := "running"
```
Comment (shenda1) [inline powerscale/provider/synciq_replication_job_resource.go:219]: Better we use constants for the states

**Review by shenda1 — APPROVED** [review]

**Comment by Krishnan-Priyanshu** [comment]
run e2e TestAccSyncIQReplicationJobResource

**Comment by Krishnan-Priyanshu** [comment]
run e2e TestAccSyncIQReplicationJob

**Comment by AnikaAgiwal2711** [comment]
run e2e TestAccSyncIQReplicationJob

**Comment by AnikaAgiwal2711** [comment]
run e2e test TestAccSyncIQReplicationJobResource

**Comment by Krishnan-Priyanshu** [comment]
run e2e test SyncIQReplicationJobResource

**Comment by Krishnan-Priyanshu** [comment]
run e2e test SyncIQReplicationJobResource

**Comment by Krishnan-Priyanshu** [comment]
run e2e test SyncIQReplicationJobResource

**Comment by Krishnan-Priyanshu** [comment]
run e2e test SyncIQReplicationJobResource

**Comment by Krishnan-Priyanshu** [comment]
run e2e test SyncIQReplicationJobResource

**Comment by Krishnan-Priyanshu** [comment]
run e2e test SyncIQReplicationJobResource

**Pull request merged**